examples/create-topic-with-revenue.js

```diff
@@ -0,0 +1,319 @@
+import { TokenCreateTransaction } from "@hashgraph/sdk";
+import { TopicUpdateTransaction } from "@hashgraph/sdk";
+import { TransferTransaction } from "@hashgraph/sdk";
+import {
+    TopicCreateTransaction,
+    TopicMessageSubmitTransaction,
+    AccountCreateTransaction,
+    AccountId,
+    PrivateKey,
+    Client,
+    Hbar,
+    CustomFixedFee,
+    AccountBalanceQuery,
+    CustomFeeLimit,
+    HbarUnit,
+} from "@hashgraph/sdk";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+async function main() {
+    if (process.env.OPERATOR_ID == null || process.env.OPERATOR_KEY == null) {
+        throw new Error(
+            "Environment variables OPERATOR_ID, and OPERATOR_KEY are required.",
+        );
+    }
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringDer(process.env.OPERATOR_KEY);
+
+    const nodes = {
+        "127.0.0.1:50211": new AccountId(3),
+    };
+
+    const client = Client.forNetwork(nodes).setOperator(
```

Comment:
`const client = Client.forName(process.env.HEDERA_NETWORK);` so the users can use it easier for other networks too. This way you won't need setting `const nodes = {...` too.